### PR TITLE
[v16] Fix Integration Button styles

### PR DESCRIPTION
**Pull request by avatus**
Backport #50258 to branch/v16

### CODE REVIEW

**Review by ravicious — APPROVED** [review]

File: web/packages/teleport/src/Integrations/IntegrationsAddButton.tsx [hunk 1]
```diff
@@ -36,6 +36,7 @@ export function IntegrationsAddButton({
 
   return (
     <HoverTooltip
+      position="bottom"
```
Comment (ravicious) [inline web/packages/teleport/src/Integrations/IntegrationsAddButton.tsx:39]: No `<FeatureHeader justifyContent="space-between">` in v16?

**Review by zmb3 — APPROVED** [review]

File: web/packages/teleport/src/Integrations/IntegrationsAddButton.tsx (hunk 1 above)
Comment (avatus) [reply web/packages/teleport/src/Integrations/IntegrationsAddButton.tsx:39]: Looking into it now

Comment (avatus) [reply web/packages/teleport/src/Integrations/IntegrationsAddButton.tsx:39]: Oh, this doesn't need to go to v16. I forgot I [fixed it in that backport](https://github.com/gravitational/teleport/pull/50246/files#diff-cc99c882ccccd7acee329b7e5f63ef2935c0e306bcfbe87c44c523fbb7611731R82) due to other conflicts. I'll close this

Comment (ravicious) [reply web/packages/teleport/src/Integrations/IntegrationsAddButton.tsx:39]: @avatus FWIW, v16 doesn't have `position="bottom"`.

https://github.com/gravitational/teleport/blob/970f4f9ae603196a92263dbd724081e2cf33fc3e/web/packages/teleport/src/Integrations/IntegrationsAddButton.tsx#L38-L48

Comment (avatus) [reply web/packages/teleport/src/Integrations/IntegrationsAddButton.tsx:39]: I'm going to ask around to understand why the recent changes to `HoverTooltip` didn't make it back to v16

**Pull request closed without merging**